core/src/serve_repair.rs

```diff
@@ -297,6 +297,28 @@ impl RepairProtocol {
             | Self::AncestorHashes { .. } => true,
         }
     }
+
+    fn max_response_packets(&self) -> usize {
+        match self {
+            RepairProtocol::WindowIndex { .. }
+            | RepairProtocol::LegacyWindowIndexWithNonce(_, _, _, _)
+            | RepairProtocol::HighestWindowIndex { .. }
+            | RepairProtocol::LegacyHighestWindowIndexWithNonce(_, _, _, _)
+            | RepairProtocol::AncestorHashes { .. }
+            | RepairProtocol::LegacyAncestorHashes(_, _, _) => 1,
+            RepairProtocol::Orphan { .. } | RepairProtocol::LegacyOrphanWithNonce(_, _, _) => {
+                MAX_ORPHAN_REPAIR_RESPONSES
+            }
```

Comment:
How often `Orphan` request do actually fill up to this max?
Wouldn't this change cause to actually unnecessarily drop `Orphan` requests which were responded previously?

Reply:
This could drop an orphan request that would have been processed previously, but would be within ~10k bytes from the current 12,000,000 byte/s limit. It doesn't seem to be worth a special case.